### PR TITLE
Allow internal_attributes in update validation templates

### PR DESCRIPTION
**Pull request by MichelSc**

### CODE REVIEW

**Comment by MichelSc** [comment]
Fix for issue https://github.com/telefonicaid/iotagent-node-lib/issues/1083

**Comment by AlvaroVega** [comment]
Could you please add an entry in CHANGES_NEXT_RELEASE about this change?

File: lib/templates/updateDevice.json [hunk 1]
```diff
@@ -177,7 +177,11 @@
         }
       }
     },
-    "static_attributes": {
+    "internal_attributes": {
+      "description": "Free form array of internal attributes for particular IOTAgents",
+      "type": "any"
+    },
+"static_attributes": {
```
Comment (mapedraza) [inline lib/templates/updateDevice.json:184]: Indentation needs to be fixed
```suggestion
    "static_attributes": {
```

**Comment by MichelSc** [comment]
entry added CHANGES_NEXT_RELEASE

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: lib/templates/updateDevice.json [hunk 2]
```diff
@@ -177,6 +177,10 @@
         }
       }
     },
+    "internal_attributes": {
+      "description": "Free form array of internal attributes for particular IOTAgents",
+      "type": "any"
```
Comment (AlvaroVega) [inline lib/templates/updateDevice.json:182]: Why any instead of array type?

Comment (MichelSc) [reply lib/templates/updateDevice.json:182]: To be aligned with the choice made in https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/templates/createDevice.json, line 197 (we must allow the same structure as the structure allowed when creating the device).

**Review by mapedraza — APPROVED** [review]
LGTM

**Review by fgalan — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**